pivy-importer/src/main/groovy/com/linkedin/python/importer/distribution/WheelsPackage.groovy

```diff
@@ -0,0 +1,157 @@
+package com.linkedin.python.importer.distribution
+
+import com.linkedin.python.importer.deps.DependencySubstitution
+import com.linkedin.python.importer.pypi.PypiApiCache
+import groovy.json.JsonSlurper
+import groovy.util.logging.Slf4j
+
+@Slf4j
+class WheelsPackage extends PythonPackage {
+
+    WheelsPackage(File packageFile,
+                  PypiApiCache pypiApiCache,
+                  DependencySubstitution dependencySubstitution,
+                  boolean latestVersions,
+                  boolean allowPreReleases,
+                  boolean lenient) {
+
+        super(packageFile, pypiApiCache, dependencySubstitution, latestVersions, allowPreReleases, lenient)
+    }
+
+    /**
+     * Get all the dependencies from package metadata. Json metadata is preferred since Version 2.0 of metadata
+     * is migrated to JSON representation. See details at https://legacy.python.org/dev/peps/pep-0426/#abstract.
+     * @return a Map whose key is configuration and value is dependencies List
+     */
+    @Override
+    Map<String, List<String>> getDependencies() {
+        Map<String, List<String>> dependenciesMap
+
+        try {
+            dependenciesMap = parseRuntimeRequiresFromMetadataJson(runtimeRequiresFromMetadataJson)
+        } catch(Exception e) {
```

Comment:
Because we don't want to error out if any problem happens with parsing metadata Json file, we have backup plan, which is parsing metadata text file.

Reply:
Seems fair to put that info in a comment.

Reply:
Yep, we have a debug log to explain that, since a lot of packages still don't have Json metadata, so we will see this message quite often, make it debug log.